lib/dynamic-cache.js

```diff
@@ -3,13 +3,15 @@
 var fs = require('fs');
 
 function objectValues(obj) {
-  if (typeof obj !== 'object') {
-    return [];
+  var values = [];
+  if (typeof obj === 'object') {
+    for (var key in obj) {
+      if (obj.hasOwnProperty(key)) {
+        values.push(obj[key]);
+      }
+    }
   }
-
-  return Object.keys(obj).map(function (key) {
-    return obj[key];
-  });
+  return values
 }
```

Comment:
The performance was pretty bad so I tried to improve it. But you're right this probably doesn't change much.